base_station/gui/src/components/CameraSelection.vue

```diff
@@ -1,6 +1,6 @@
 <template>
   <div class="wrap2">
-    <span v-if="cam_index >= 0" class="title">Current Camera: {{cam_index}}</span>
+    <span v-if="cam_index >= 0" class="title">Current Camera: {{cam_index - 1}}</span>
     <div class="buttons">
       <template v-for="i in 3">
         <button class="cam_buttons" ref="cams" v-on:click="$emit('cam_index', i-1)"> <span>{{cameras[i-1]}}</span> </button>
```

Comment:
change to i - 2 so these cameras have indices -1, 0, and 1. do similar things for the rest.